README.md

```diff
@@ -27,74 +38,59 @@ sudo apt-get install wget
 
 Note: You can check other available options for downloading `wget` [here](https://www.gnu.org/software/wget/faq.html#download).
 
-**Get Source Code**
+#### Install binaries
 
-```
-go get github.com/cosmos/cosmos-sdk
-```
-
-Now we can fetch the correct versions of each dependency by running:
+Cosmos SDK can be installed to `$GOPATH/src/github.com/cosmos/cosmos-sdk` like a normal Go program:
 
 ```
+go get github.com/cosmos/cosmos-sdk
 cd $GOPATH/src/github.com/cosmos/cosmos-sdk
 git fetch --all
-git checkout 0f2aa6b
+git checkout d613c2b9
 make get_tools // run $ make update_tools if already installed
 make get_vendor_deps
 make install
 ```
 
-The latest binaries should now be installed. Verify that everything is OK by running:
+This will install `gaiad` and `gaiacli` and four example binaries: `basecoind`, `basecli`, `democoind`, and `democli`. Verify that everything is OK by running:
 
 ```
 gaiad version
-```
-
-You should see:
-
-```
-0.15.0-rc0-0f2aa6b
-```
-
-And also:
-
-```
 gaiacli version
 ```
 
-You should see:
+You should see in both cases:
 
 ```
-0.15.0-rc0-0f2aa6b
+0.15.0-rc0-d613c2b9
 ```
 
-## Genesis Setup
+### Genesis Setup
+
+Now that we have completed the basic SDK setup, we can start working on the genesis configuration for the chain we want to connect to.
 
 Initiliaze `gaiad` :
 
 ```
 gaiad init
 ```
 
-You can find the corresponding genesis files [here](https://github.com/tendermint/testnets). Then replace the `genesis.json`and `config.toml` files:
+You can find the corresponding genesis files [here](https://github.com/cosmos/testnets). Then replace the `genesis.json` and `config.toml` files:
 
 ```
-rm $HOME/.gaiad/config/genesis.json $HOME/.gaiad/config/config.toml $HOME/.gaiad/config/addrbook.json
-
 wget -O $HOME/.gaiad/config/genesis.json https://raw.githubusercontent.com/cosmos/testnet/master/gaia-4000/genesis.json
 
 wget -O $HOME/.gaiad/config/config.toml https://raw.githubusercontent.com/cosmos/testnet/master/gaia-4000/config.toml
```

Comment:
Are we going to update this to the new testnet? I guess as soon as we release it - we should update simultaneously

Reply:
Idk yet which will be the name for the new testnet...

Reply:
I think we're settting up gaia5000 today